packages/customize-widgets/src/components/header/style.scss

```diff
@@ -1,3 +1,19 @@
+.customize-widgets-header {
+	position: sticky;
+	top: 0;
+
+	// Offset the customizer's sidebar padding.
+	margin: -15px ( -$grid-unit-15 ) 0 ( -$grid-unit-15 );
+	padding: $grid-unit-15;
+
+	// Match the customizer grey background.
+	background: #f0f0f1;
+	border-bottom: 1px solid $gray-200;
+
+	// Ensure widget UI doesn't overlap when scrolling
+	z-index: 8;
```

Comment:
What's the reason for 8 here? 2 seems to work as well. Might be worth adding something to the comment about what we're trying to overlay.

Reply:
I've move this to the z-indexes file and added a better comment 👍 

Also split the topbar and toolbar indices. I figure it's best to make them as high as possible as nothing else should overlap them other than the odd element (popovers, tooltips) that already has a much higher z-index.